Sources/Core/Operators.swift

```diff
@@ -74,11 +76,28 @@ infix operator ||: LogicalDisjunctionPrecedence
 
 infix operator +: AdditionPrecedence
 infix operator -: AdditionPrecedence
+infix operator |: AdditionPrecedence
+infix operator ^: AdditionPrecedence
 
 infix operator &: MultiplicationPrecedence
 infix operator *: MultiplicationPrecedence
+infix operator /: MultiplicationPrecedence
+infix operator %: MultiplicationPrecedence
 
 infix operator &=: AssignmentPrecedence
 infix operator +=: AssignmentPrecedence
 infix operator -=: AssignmentPrecedence
 infix operator *=: AssignmentPrecedence
+infix operator /=: AssignmentPrecedence
+infix operator %=: AssignmentPrecedence
+infix operator |=: AssignmentPrecedence
+infix operator ^=: AssignmentPrecedence
+infix operator <<=: AssignmentPrecedence
+infix operator &<<=: AssignmentPrecedence
+infix operator >>=: AssignmentPrecedence
+infix operator &>>=: AssignmentPrecedence
```

Comment:
What do you think of splitting up the bit-shift and assign and ignoring-overflow variants into separate groups?